xsd/siri.xsd

```diff
@@ -361,6 +361,11 @@ Rail transport, Roads and road transport.
   </xsd:annotation>
   <xsd:sequence>
    <xsd:group ref="ServiceDeliveryRequestStatusGroup"/>
+   <xsd:element name="AllData" type="xsd:boolean" default="false" minOccurs="0">
```

Comment:
Christophe: Shouldn't the default be true? In a Response pattern the default should be true.

Reply:
need to avoid a default which results in all data being supplied each time. All data should only be sent when requested otherwise it could be very inefficient.